odoo/tests/common.py

```diff
@@ -950,6 +950,20 @@ def _reset(cb, funcs, data):
         self.cr.execute('SAVEPOINT test_%d' % self._savepoint_id)
         self.addCleanup(self.cr.execute, 'ROLLBACK TO SAVEPOINT test_%d' % self._savepoint_id)
 
+    @contextmanager
+    def enter_registry_test_mode(self):
```

Comment:
It feels strange to start the name of a context manager with "enter" since a context manager is both an enter and an exit.